src/Integration.Vsix/Settings/GeneralOptionsDialogControl.xaml

```diff
@@ -39,29 +35,8 @@
                    VerticalAlignment="Center"
                    Margin="5,8,0,5" TextWrapping="Wrap" Text="Note: per-user settings are ignored for solutions using connected mode." />
         </StackPanel>
-        <Button Grid.Column="0" Grid.Row="2" Content="Activate more..." Click="OnActivateMoreClicked"
```

Comment:
Delete all of the unnecessary rows and put the verbosity panel in the new last row.